js/class_add_remove.js

```diff
@@ -79,7 +79,7 @@ class BookCollection {
     }
   }
 
-  removeBook(data) {
+  removebook(data) {
     const title = data.id.replace(/_/g, ' ');
```

Comment:
- When i try to remove a book from the list. Appears an error in console saying that your data it's undefined.

Reply:
Thanks for pointing it out. I fixed this problem by turning into a string every numerical value that is input as a title. As well as fixing the id designation when the title includes non-alphanumerical characters. Now it works correctly.